src/constructor.io/utils/Constants.cs

```diff
@@ -8,6 +8,7 @@ public static class Constants
         public const string CONSTRUCTOR_TOKEN = "constructorToken";
         public const string FACET_NAME = "facet_name";
         public const string FILTERS = "filters";
+        public const string FILTERS_PER_SECTION = "filters_per_section";
```

Comment:
I think we normally only keep valid query pareters or request headers in this constants file, but I can kind of see why we did it this way.